Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:16.04
+ADD . /ardupilot
+WORKDIR /ardupilot
+
+RUN apt-get update && apt-get upgrade -y && apt-get dist-upgrade -y && apt-get install lsb-release sudo software-properties-common python-software-properties -y
+ENV USER=root
+RUN bash -c "Tools/scripts/install-prereqs-ubuntu.sh -y && apt-get install gcc-arm-none-eabi -y"
```

Comment:
why use gcc-arm-none-eabi ? the install script should already have install the right toolchain !

Reply:
I thought that as well, watching the script. Without this, gcc claimed it couldn't find it.

Reply:
because you need to source the .profile file or at the toolchain location on the path !

Reply:
Quick note, AFAIK install-prereqs-ubuntu.sh tarball puts only a 32 bit version of gcc-arm-none-eabi to /opt not the 64bit version. Correct me if I'm wrong...